src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -335,4 +335,35 @@ export function isTCGAProvStudy(studyId:string){
 
 export function isPanCanStudy(studyId:string){
     return /tcga_pan_can_atlas/.test(studyId);
+}
+
+
+export function buildResultsViewPageTitle(genes:string[], studies:CancerStudy[]){
```

Comment:
it may not be worth changing, for such an inert and single-purpose function, but I find this pretty hard to read, like to figure out what it will end up looking like. It may be a lot easier to get by using template strings.

Reply:
i hear ya.  but i find it difficult doing conditional logic with templates.  like, you end up having to do lots of concatenation anyway.  spaces and such.